analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataReference.cs

```diff
@@ -156,6 +156,7 @@ internal static class NuGetMetadataReference
         public static References SystemTextRegularExpressions(string packageVersion = "4.3.1") => Create("System.Text.RegularExpressions", packageVersion);
         public static References SystemThreadingTasksExtensions(string packageVersion) => Create("System.Threading.Tasks.Extensions", packageVersion);
         public static References SystemValueTuple(string packageVersion) => Create("System.ValueTuple", packageVersion);
+        public static References TimeZoneConverter(string packageVersion = Constants.NuGetLatestVersion) => Create("TimeZoneConverter", packageVersion);
```

Comment:
For the very same reasons you mentioned here, I opted and would opt for the latest version.
In my opinion we want to know if there were changes in the package. I agree that it is painful when all of a sudden it fails and in the middle of something else you are doing you need to do the investigation to make the pipeline green again, and at the same time this can be a heads-up for us to see if the rule needs adaptation.